src/main/java/com/mycompany/exercisetwo/Cow.java

```diff
@@ -0,0 +1,47 @@
+package com.mycompany.exercisetwo;
+
+import java.time.LocalDate;
+import java.time.Period;
+
+/**
+ *
+ * @author Cristian Gallego
+ */
+public class Cow extends Mammalian implements Death {
+
+    public String result;
+
+    @Override
+    public String die(LocalDate born, LocalDate die, String deathCause) {
+        
+        Period period = Period.between(born, die);
+
+        if (period.getYears() % 2 == 0) {
+
+            result = "pair number";
+        } else {
+            result = "odd number";
+        }
+        return "The date of death was: " + die + ", the number is " + result + ", lived: " + period.getYears() + " years, and the death cause was: " + deathCause;
```

Comment:
You can improve this logic in "result" by using something called ternary operator

https://www.baeldung.com/java-ternary-operator
https://www.geeksforgeeks.org/java-ternary-operator-with-examples/

Reply:
Also remember the exercise says "Morir, debe calcular la edad y **multiplicarla por dos"**

Reply:
Done, ternary operator used and calculate corrected.